packages/hardhat-plugin/test/execution/timeouts/deploy-run-times-out.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/no-unused-modules */
+import { defineModule } from "@ignored/ignition-core";
+import { assert } from "chai";
+
+import {
+  TestChainHelper,
+  useDeploymentDirectory,
+} from "../nonce-checks/useDeploymentDirectory";
+
+/**
+ * A run that deploys a contract times out
+ */
+describe("execution - deploy run times out", () => {
+  useDeploymentDirectory("minimal-new-api", "deploy-run-times-out");
+
+  it("should error naming timed out transactions", async function () {
```

Comment:
This timeout test leverages not processing a block.

Ideally I would like to add a further test where blocks are continuously processed but the transactions times out because the transaction is not picked up individually.

@alcuadrado is gas limit the best way of doing this?

Reply:
That's an easy way to achieve it, yes.